sentry_sdk/integrations/django/caching.py

```diff
@@ -20,6 +20,18 @@
 ]
 
 
+def _get_span_description(method_name, args, kwargs):
+    # type: (str, Any, Any) -> str
+    description = "{} ".format(method_name)
+
+    if args is not None and len(args) >= 1:
+        description += text_type(args[0])
+    elif kwargs is not None and "key" in kwargs:
+        description += kwargs["key"]
```

Comment:
Would it make sense to run `kwargs["key"]` through `text_type` as well? I imagine `kwargs["key"]`, if present, _should_ be a string but maybe would be worth it make sure.